frontend/app/bookshelf-login/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Button } from 'components/buttons/Button'
+import { Input } from 'components/forms/Input'
+import type { NextPage } from 'next'
+import Image from 'next/image'
+
+const LoginContainer: NextPage = () => {
+  return (
+    <div className="bg-background flex h-screen w-full justify-center">
```

Comment:
How are you finding tailwind?

Reply:
Feels like going back in the days of building styles from scratch. Whereas using Chakra, if I need a certain component say `Modal` it's already created.

Reply:
Feels like going back in the days of building styles from scratch. Whereas using Chakra, if I need a certain component say `Modal` it's already created, less work to be honest. So if it's not that design intensive or rush project, Chakra or alike is a go.

What I like about Tailwind is the clean design system principle. Using Tailwind, you need to have the grasp of semantic html. Also, I feel I own the design using Tailwind. hahaha. 👌